en/4-blocks/2-Structure_of_a_block.md

```diff
@@ -0,0 +1,146 @@
+
+	Neo however is using [dBFT](https://docs.neo.org/en-us/basic/consensus/whitepaper.html), which makes a block final with as little as 1 confirmation, currently in ~15 seconds. For a hacker to get a malicious block to replace a valid block, he would need to convince 66% of the consensus nodes to build the next block on top of his malicious blockchain, and use his malicious block header as the 'previous block hash' in the new block.
+
+- ***merkleroot***: The hash of the root of the Merkle Tree of the block' transactions.
+
+- ***time***: The block time is a Unix epoch time of the moment the block was created. A Unix Epoch time is the amount of seconds since Epoch - January 1, 1970 at 00:00:00 GMT.
+
+- ***index***: The height of the current block. The Genesis block has index 0. This field is not present in the block itself but calculated by the node serving the RPC request for completeness.
+
+- ***Nonce***: A random number that can be used as such by a Smart Contract. It should be understood that this is quite different from the nonce in on other blockchain protocols like for example Bitcoin, where miners compete to find the nonce that satisfies a certain difficulty level.  Because of the dBFT consensus mechanism in Neo, there is no Proof of Work (PoW) and nodes are working together to achieve consensus. Since there is no PoW required, the Nonce field is used as a random number to be used by all transactions occurring in that block. When each block is generated, the consensus nodes will reach a consensus on a random number, and fill it into the Nonce field of the new block. The contract program can easily obtain the random number of any block, by referencing the Nonce field.
+
+- ***nextconsensus***: During consensus on the current block, the algorithm ensures a new consensus node is chosen, which is stored in this field.
+
+- ***script***: Script used to validate the block. Discussed in detail in [Block Validation & Processing](4-Block_validation_processing.md).
+
+## Block body
+
+
+The body of a block contains all the transactions that are embodied in that block.
+
+### Transaction types
+Below are the type of transactions that can occur in a block:
+* **MinerTransaction**: The first transaction in each block must be MinerTransaction. It is used to reward all transaction fees of the current block to the validator.
+* **IssueTransaction**: Used to issue an asset. Asset managers can create the assets that have been registered in NEO's block chain through IssueTransaction, and sent them to any address.
```

Comment:
sent them to any address ->
send them to any address

Reply:
fixed, will push with other fixes.